.github/workflows/lint.yml

```diff
@@ -16,12 +16,26 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
-          cache: false
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
           version: v1.57
 
+  goimports:
```

Comment:
I don't think we need this.
golangci already enabled the goimports checks

Reply:
It's not failing the lint for failed goimports.

Reply:
Can you at least move it to Makefile target?

Reply:
I figured it out. I configured golangci to get the desired behaviour and removed the new action.